examples/metrics_datadog/sam/template.yaml

```diff
@@ -0,0 +1,31 @@
+AWSTemplateFormatVersion: "2010-09-09"
+Transform: AWS::Serverless-2016-10-31
+Description: Powertools for AWS Lambda (Python) version
+
+Globals:
+  Function:
+    Timeout: 5
+    Runtime: python3.10
+    Tracing: Active
+    Environment:
+      Variables:
+        POWERTOOLS_METRICS_NAMESPACE: ServerlessAirline
+        DD_API_KEY: "<YOUR DATADOG API KEY>"
+        DD_SITE: datadoghq.com
+
+    Layers:
+      # Find the latest Layer version in the official documentation
+      # https://docs.powertools.aws.dev/lambda/python/latest/#lambda-layer
+      - !Sub arn:aws:lambda:${AWS::Region}:017000801446:layer:AWSLambdaPowertoolsPythonV2:40
+      # Find the latest Layer version in the Datadog official documentation
+      # See https://github.com/DataDog/datadog-lambda-python/releases
+      - !Sub arn:aws:lambda:${AWS::Region}:464622532012:layer:Datadog-Python310:78
+      # See https://github.com/DataDog/datadog-lambda-extension/releases
+      - !Sub arn:aws:lambda:${AWS::Region}:464622532012:layer:Datadog-Extension:45
```

Comment:
Please add a comment on what these mean for a newcomer. One good practice in docs is to avoid a link hell, where they need to jump to multiple places to get the info we might give them upfront (if they need more details, sure!)